src/App.js

```diff
@@ -1,26 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
-    <div className="App">
```

Comment:
- [ ] It would be better if you make your App.js class-based because this is current milestone's one of the purposes.